packages/builders/src/cypress/schema.json

```diff
@@ -30,6 +30,11 @@
       "type": "string",
       "description":
         "Use this to pass directly the address of your distant server address with the port running your application"
+    },
+    "browser": {
+      "type": "string",
```

Comment:
Please add an `enum` to list the allowed types of browsers. (Users unfamiliar with Cypress will expect other browsers such as `firefox` and `safari` to be available which is not yet the case.

https://docs.cypress.io/guides/guides/launching-browsers.html#Browsers

```json
"enum": [
  "electron",
  "chrome",
  "chromium",
  "canary"
]
```